src/test/multiRootWkspc/disableLinters/file.py

```diff
@@ -0,0 +1,87 @@
+"""pylint option block-disable"""
```

Comment:
Probably should either (a) make this work under Python 3, or (b) add a comment that this code is actually never run so it shouldn't effect if Python 2 or 3 is used. (Same for other test files).

Reply:
Will do

Reply:
Fixed